Makefile.BSD

```diff
@@ -14,7 +14,14 @@ ${D}/Makefile::
 	( cd ${D} && printf 'SRCS=' && ${MAKE} -V SRCS | sed -e 's| cpusupport-config.h||' ) >> $@
 	( cd ${D} && printf 'IDIRS=' && ${MAKE} -V IDIRS ) >> $@
 	( cd ${D} && printf 'LDADD_REQ=' && ${MAKE} -V LDADD_REQ ) >> $@
-	cat Makefile.prog >> $@
+	# Copy the Makefile template, removing the "install:"
```

Comment:
Normally I like comments... but makefile parsers are finicky things, and I would be completely unsurprised if a comment here caused problems for some versions of make.

Actually, I *think* this would simply get passed through to the shell, which would interpret it as a comment... but still, let's not take chances.